### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mdchristians**
Cleanup and reorganize to slim the package size and eliminate duplicated dependencies.

### CODE REVIEW

File: src/theme/base.scss [hunk 1]
```diff
@@ -111,6 +117,49 @@ cite {
...
+// Forms
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+
+label {
+  display: block;
+  font-size: 0.9em;
```
Comment (mariadoesmath) [inline src/theme/base.scss:133]: Curious where these styles came from? Did you write them? Did we copy them from the main app?

File: src/theme/index.scss [hunk 2]
```diff
@@ -1,2 +1,17 @@
 @import 'variables';
```
Comment (mariadoesmath) [inline src/theme/index.scss:1]: Wow, I just looked at the dist css file and I totally did not realize we were resetting the same styles all over the place. This is a great improvement! Thanks Matt!!!!!

**Comment by mariadoesmath** [comment]
Storybook is not looking so hot. Looks like the styles aren't being applied.

**Comment by mdchristians** [comment]
@mariadoesmath yeah i saw that.  I'm looking into it.

**Pull request merged**